src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -74,7 +80,13 @@ public RsWithCookie(final Response res, final String name,
         final String value, final String... attrs) {
         super(
             new RsWithHeader(
-                res, "Set-Cookie", RsWithCookie.make(name, value, attrs)
+                res,
+                "Set-Cookie",
+                RsWithCookie.make(
+                    RsWithCookie.checkName(name),
+                    RsWithCookie.checkValue(value),
+                    attrs
+            )
```

Comment:
@dmzaytsev hmm let's leave it. that way when qulice gets fixed we'll be notifified by a build failure. otherwise we'll forget about the indentation suppresion.